index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title>Portfolio: Setup and Mobile First</title>
+</head>
+<body>
+    <main class="header-background-mobile">
+        <header class="toolbar">
+            <nav>
+                <div class="mylogo">
+                    <a href="#">My Logo</a>
+                </div>
+                <div class="union">
+                    <img src="./images/Union.png" alt="Hamburguer Menu">
+                </div>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="primarytext">
+                <h1>I'm Federico Glad to see you!</h1>
+            </div>
+            <div class="bodytext">
+                <p>I'm a software developer! I can help you build a product , feature or 
+                    website Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+            </div>
+            <div class="socialmedia">
+                <div id="supportingtext">
+                    <h5>LET'S CONNECT</h5>
+                </div>
+                <div>
+                    <ul class="socialmediaicons">
+                        <li><img src="./images/Vector.svg" alt="Twitter"></li>
+                        <li><img src="./images/Linkedinicon.svg" alt="Linkedin"></li>
+                        <li><img src="./images/Vector1.svg" alt="Mail"></li>
+                        <li><img src="./images/Vector2.svg" alt="GitHub"></li>
+                        <li><img src="./images/Vector3.svg" alt="Peace"></li>
```

Comment:
- [ ] As required for the project, kindly make sure each icon has a link (`<a>` tag) apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using the `href` attribute.
i.e: 
```HTML
<li>
  <a><img src="./images/Vector.svg" alt="Twitter"></a>
</li>

<li>
  <a><img src="./images/Linkedinicon.svg" alt="Linkedin"></a>
</li>
...
```